action/envelope.go

```diff
@@ -269,8 +225,5 @@ func (elp *envelope) LoadProto(pbAct *iotextypes.ActionCore) error {
 	return nil
 }
 
-// SetNonce sets the nonce value
-func (elp *envelope) SetNonce(n uint64) { elp.nonce = n }
-
 // SetChainID sets the chainID value
 func (elp *envelope) SetChainID(chainID uint32) { elp.chainID = chainID }
```

Comment:
also removed?

Reply:
`SetChainID ` is not called on `AbstractAction` but called by `Envelope`, so it's fine to keep it here